server/webapp/WEB-INF/vm/build_detail/_build_detail_summary_jstemplate.vm

```diff
@@ -29,7 +29,11 @@
     <span>Not yet assigned</span>
   {/if}
   #if ( $elasticProfilePluginId && $userHasAdministratorRights )
-    <a href="$req.getContextPath()/admin/status_reports/$elasticProfilePluginId" class="btn-primary btn-small status-report-btn-small">Check Plugin Status</a>
+    {if build.build_assigned_date != -1}
```

Comment:
@gocd/committers --- With this implementation if plugin has not implemented `agent-status-report` call it will raise `UnhandledRequestTypeException`.  We will update the document saying both the call is mandatory if plugin supports [status_report](https://github.com/gocd/gocd/blob/master/plugin-infra/go-plugin-domain/src/main/java/com/thoughtworks/go/plugin/domain/elastic/Capabilities.java#L26) capability. Is this okay ?